modules/core/src/test/java/org/apache/ignite/util/GridCommandHandlerTest.java

```diff
@@ -253,6 +260,72 @@ public void testDeactivate() throws Exception {
         assertFalse(ignite.cluster().active());
     }
 
+    /**
+     * Test the deactivation command on the active and no cluster with checking
+     * the cluster name(which is set through the system property) in
+     * confirmation.
+     *
+     * @throws Exception If failed.
+     * */
+    @Test
+    @WithSystemProperty(key = IGNITE_CLUSTER_NAME, value = CLUSTER_NAME)
+    public void testDeactivateWithCheckClusterNameInConfirmationBySystemProperty() throws Exception {
+        IgniteEx igniteEx = startGrid(0);
+        assertFalse(igniteEx.cluster().active());
+
+        String clusterName = CLUSTER_NAME;
+
+        deactivateWithCheckClusterNameInConfirmation(igniteEx, clusterName);
+
+        igniteEx.cluster().active(true);
+        assertTrue(igniteEx.cluster().active());
+
+        deactivateWithCheckClusterNameInConfirmation(igniteEx, clusterName);
+    }
+
+    /**
+     * Test the deactivation command on the active and no cluster with checking
+     * the cluster name(default) in confirmation.
+     *
+     * @throws Exception If failed.
+     * */
+    @Test
+    public void testDeactivateWithCheckClusterNameInConfirmationByDefault() throws Exception {
+        IgniteEx igniteEx = startGrid(0);
+        assertFalse(igniteEx.cluster().active());
+
+        String clusterName = igniteEx.context().cache().utilityCache().context().dynamicDeploymentId().toString();
+
+        deactivateWithCheckClusterNameInConfirmation(igniteEx, clusterName);
```

Comment:
Move common logic (lines: 299-304) in both tests (`testDeactivateWithCheckClusterNameInConfirmationBySystemProperty` and `testDeactivateWithCheckClusterNameInConfirmationByDefault`) in private method.